package.json

```diff
@@ -298,6 +298,7 @@
   },
   "workspaces": [
     "packages/client",
-    "packages/types"
+    "packages/types",
+    "packages/mattermost-components"
```

Comment:
Perhaps this path should be `packages/components` to match the `@mattermost/components` name?

Reply:
Also, this is a chance to a different part of this file, but we probably need to add the new package to Jest's `moduleNameMapper` above so that it can find the files while running unit tests

Reply:
I think `mattermost-components` better matches `@mattermost/components` than just `components` but 0/5

I don't think we should because it's a real compiled package.

Reply:
Ok will change.